Assets/MaxPecora/Project 4/Scripts/DestroyFallingObject.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace max
+{
+
+	public class DestroyFallingObject : MonoBehaviour
+	{
+		public GameObject explosion;
+
+		void OnTriggerEnter2D(Collider2D col)
+		{
+			if (col.gameObject.name.Equals("ball"))
+			{
+				GameObject[] boxes = GameObject.FindGameObjectsWithTag("box");
+				for (int i = 0; i < boxes.Length; i++)
+				{
+					Destroy(boxes[i]);
+				}
+				GameObject.Find("gameOverMenu").GetComponent<Transform>().localScale = new Vector2(1, 1);
```

Comment:
Consider moving all this behavior to some other gameobject that you can reference directly instead of doing `Find()` calls here. It'll simplify it and make it easier to read.